integration-tests/testconfig/log_poller/example.toml

```diff
@@ -69,6 +69,88 @@ chain_id=1337
 # list of addresses to be prefunded in genesis
 addresses_to_fund=["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"]
 
+[PrivateEthereumNetwork.EthereumChainConfig.HardForkEpochs]
+# hardforks to be applied (fork_name = epoch)
+Deneb=500
+
+# Chainlink node TOML config
+[NodeConfig]
+BaseConfigTOML = """
```

Comment:
Do we have to nest this TOML? Depending on the TOML lib, I think it would be possible to use a type for this field which defers parsing (like `json.RawMessage`), so that we could handle it as dynamic TOML instead of an opaque string.

Reply:
I do like this idea, but my only doubt is that it will work differently than it does now in CCIP and idk if it's worth diverting.